### ISSUE TITLE
[TF2] Reduce Ghost Fort/Lakeside event countdown per team to 3 min

### ISSUE THREAD

**Issue by Ashetf2** [ValveSoftware/Source-1-Games#4348]
7 minutes per team makes the map to last too long, considering that Merasmus appears so often, and makes obtaining the "A Lovely Vacation Spot" achievement and completing contracts in this map very frustasting to do.

**Comment by condor00fr** [comment]
prehaps five instead? so that Merasmus can still spawn more that once?

**Comment by MikhailaTF** [comment]
Whether it's reduced to 3 minutes as originally suggested or 5 as per @condor00fr's comment can be debated, but if something needs to be changed before/in this year's halloween update it'll be this! Much desired QoL change!

**Comment by Ashetf2** [comment]
> prehaps five instead? so that Merasmus can still spawn more that once?

I fond 5 min still too long, and no, Merasmus can appear more than once with a 3 min timer

**Comment by ShadowMan44** [comment]
Maybe they could also make it so Merasmus takes about maybe... 30 seconds longer to respawn each time? That way he won't keep popping up so often.

**Comment by Whurrhurr** [comment]
remove the mechanic that makes merasmus spawn 1 minute faster every time you kill him until his respawn timer becomes 1 minute and then make the round timer 3 minutes. merasmus spawning faster because you killed him is a dumb punishment for beating the thing that grinds the game to a halt for 2 minutes so you can get back to playing the game again.

**Comment by FrozenDragon0** [comment]
Yes please reduce the time on this map, half of the time this map can take up to 30/45 minutes total just to end the game fully due to Merasmus spawning in so frequently.

**Comment by Ashetf2** [comment]
Please take in consideration this request for this halloween :(

**Comment by jh34ghu43gu** [comment]
> remove the mechanic that makes merasmus spawn 1 minute faster every time you kill him until his respawn timer becomes 1 minute and then make the round timer 3 minutes. merasmus spawning faster because you killed him is a dumb punishment for beating the thing that grinds the game to a halt for 2 minutes so you can get back to playing the game again.

It's (probably) not meant to be a punishment, it's probably meant to help get a higher skull topper level faster for those that grind it. The contracts that were added later have unfortunately conflicted with this goal. Allowing halloween contracts to be finished without waiting for the round to end would probably be a better solution (#4403 related).

**Comment by agrastiOs** [comment]
I assume the longer timer was also intended for the time when there was no truce - to actually give a chance for people to kill Merasmus who didn't have chance to do so yet to get the skull topper.

Now that we have the truce which stops everything and forces you to fight the boss, the rounds take way too much time as people kill Merasmus often and he respawns yet more. This is thanks to the truce that was implemented for Scream Fortress 2015 and to let people fullfill the Merasmus and Monoculus contracts more easily. The ConTracker system (2017) forcing people to stay for the entire round to complete the round worsens the problem even further.

Obviously this issue is only even apparent due to the compounding problem that several updates to the contract systems and bosses created, and this design issue wasn't apparent originally. So I would agree that Ghost Fort needs to have its timer reduced - even if they made it so you could turn in contracts and leave earlier, staying for an entire match takes so long that it's rarely fun to play one.
